awx/ui_next/src/screens/Inventory/SmartInventoryHosts/SmartInventoryHostList.test.jsx

```diff
@@ -12,125 +12,107 @@ import mockHosts from '../shared/data.hosts.json';
+  });
 
-    test('should enable run commands button when at least one host is selected', () => {
-      act(() => {
-        wrapper.find('DataListCheck[id="select-host-2"]').invoke('onChange')(
-          true
-        );
-      });
-      wrapper.update();
-      const runCommandsButton = wrapper.find(
-        'button[aria-label="Run commands"]'
-      );
-      expect(runCommandsButton.prop('disabled')).toEqual(false);
-    });
+  afterAll(() => {
+    jest.clearAllMocks();
+    wrapper.unmount();
+  });
 
-    test('should select and deselect all items', async () => {
-      act(() => {
-        wrapper.find('DataListToolbar').invoke('onSelectAll')(true);
-      });
-      wrapper.update();
-      wrapper.find('DataListCheck').forEach(el => {
-        expect(el.props().checked).toEqual(true);
-      });
-      act(() => {
-        wrapper.find('DataListToolbar').invoke('onSelectAll')(false);
-      });
-      wrapper.update();
-      wrapper.find('DataListCheck').forEach(el => {
-        expect(el.props().checked).toEqual(false);
-      });
+  test('initially renders successfully', () => {
+    expect(wrapper.find('SmartInventoryHostList').length).toBe(1);
+  });
+
+  test('should fetch hosts from api and render them in the list', () => {
+    expect(InventoriesAPI.readHosts).toHaveBeenCalled();
+    expect(wrapper.find('SmartInventoryHostListItem').length).toBe(3);
+  });
+
+  test('should have run command button', () => {
+    wrapper.find('DataListCheck').forEach(el => {
+      expect(el.props().checked).toBe(false);
     });
+    const runCommandsButton = wrapper.find('button[aria-label="Run command"]');
+    expect(runCommandsButton.length).toBe(1);
+  });
 
-    test('should show content error when api throws an error', async () => {
-      InventoriesAPI.readHosts.mockImplementation(() =>
-        Promise.reject(new Error())
-      );
-      await act(async () => {
-        wrapper = mountWithContexts(
-          <SmartInventoryHostList inventory={mockInventory} />
-        );
-      });
-      await waitForElement(wrapper, 'ContentError', el => el.length === 1);
+  test('should select and deselect all items', async () => {
+    act(() => {
+      wrapper.find('DataListToolbar').invoke('onSelectAll')(true);
+    });
+    wrapper.update();
+    wrapper.find('DataListCheck').forEach(el => {
+      expect(el.props().checked).toEqual(true);
+    });
+    act(() => {
+      wrapper.find('DataListToolbar').invoke('onSelectAll')(false);
+    });
+    wrapper.update();
+    wrapper.find('DataListCheck').forEach(el => {
+      expect(el.props().checked).toEqual(false);
     });
   });
 
-  describe('User does not have adhoc permissions', () => {
-    let wrapper;
-    const clonedInventory = {
-      ...mockInventory,
-      summary_fields: {
-        user_capabilities: {
-          adhoc: false,
-        },
-      },
-    };
+  test('should render enabled ad hoc commands button', async () => {
```

Comment:
This test could probably be merged with the test "should have run command button" on line 74.